rkt/api_service.go

```diff
@@ -289,26 +283,27 @@ func getNetworks(p *pod) []*v1alpha.Network {
 func getBasicPod(p *pod) (*v1alpha.Pod, *schema.PodManifest, error) {
```

Comment:
And in InspecdPod() API, if you give a non-existent ID, it will return an error.